client/blocks/cs-embed/edit.js

```diff
@@ -105,7 +105,7 @@ const EmbedForm = ( { attributes, setAttributes } ) => {
 			) : (
 				<Placeholder
 					icon={ CSLogo }
-					label={ __( 'Survey Embed', 'crowdsignal-forms' ) }
+					label={ placeholderTitle }
```

Comment:
This isn't necessarily wrong but I do wonder if it's really the best way to keep these things inside block attributes 🤔.  
I think a potentially better way would be to create a separate component for the placeholder and define a map of different props to use based on what the current variation is.

Really, it kind of works the same way except it's a separate object and doesn't pollute `attributes` with _random data_ that's really not used for anything ever again once the block is created.
I'm not sure it's worth changing this now but do keep it in mind in the future :)